source/GameAction.cpp

```diff
@@ -226,6 +254,29 @@ void GameAction::LoadSingle(const DataNode &child)
 		fail.insert(child.Token(1));
 	else if(key == "fail")
 		failCaller = true;
+	else if(key == "attributes")
+	{
+		// Current expected format is attributes add/set <attribute name> <value>
+		if(child.Size() == 4)
+		{
+			if(child.Token(1) == "add")
+			{
+				double valueChange = static_cast<double>(child.Value(3));
```

Comment:
The static cast icks me a bit, is it necessary?

Reply:
I don't know, actually. It worked, so there it is. Either the code came from VitalChip; or I saw it used somewhere in ES else to get the value into a double, so I reused it.

Reply:
I think it's supposed to be a double already
You can hover over the function Value to check btw

Reply:
Thanks!

Reply:
Hmmm, what do I need to hover over to see the what sort of variable it is? And I suppose, what program are you using? 

I'm in VS 2022, and I am not seeing anything pop up when I hover.